src/atoms/Flex/__tests__/Flex.native.test.js

```diff
@@ -1,15 +1,140 @@
 import React from 'react';
 import { render } from '@testing-library/react-native';
-import { View } from 'react-native';
+
+import View from '../../View';
 import Flex from '../index';
+import Space from '../../Space';
+import Size from '../../Size';
+
+const defaultFlexStyles = { flexDirection: 'column' };
+const defaultSpaceStyles = {};
+const defaultSizeStyles = {};
 
 describe('Renders <Flex /> correctly', () => {
-  it('snapshot testing', () => {
-    const { container } = render(
+  it('renders child of <Flex />(no styles) correctly', () => {
+    const { container, getByTestId } = render(
       <Flex>
-        <View />
+        <View testID={'flex-view'} />
+      </Flex>,
+    );
+    const childView = getByTestId('flex-view');
+    expect(childView.props.style).toMatchObject([defaultFlexStyles]);
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('<Flex /> and <Space /> together plays properly', () => {
+  it('renders child of <Flex />(no styles) and <Space />(no styles) correctly', () => {
```

Comment:
```suggestion
  it('renders <Space /> as child component of <Flex />with expected props', () => {
```

Reply:
Not just Flex as child of Space but it mainly tests the **View** which is a child of Flex which in turn is a child of Space has **received correct styles**.

Reply:
Not just Flex as child of Size but it mainly tests the **View** which is a child of Flex which in turn is a child of Size has **received correct styles**.